### PR TITLE
refactor/multi collaterals on initialization

### PR DESCRIPTION
**Pull request by evandrosaturnino**

### CODE REVIEW

File: packages/lib-ethers/utils/deploy.ts [hunk 1]
```diff
@@ -308,97 +280,248 @@ const connectContracts = async (
...
+        async ([addresses, startBlock]) => {
+          
+          const thusdToken = await deployContract(
+            deployer,
+            getContractFactory,
+            "THUSDToken",
+            firstDeployment.addresses.troveManager,
+            firstDeployment.addresses.stabilityPool,
+            firstDeployment.addresses.borrowerOperations,
+            addresses.troveManager,
```
Comment (vzotova) [inline packages/lib-ethers/utils/deploy.ts]: naming a bit confusing, why some called `firstDeployment.addresses` and other just `addresses`?

Comment (evandrosaturnino) [reply packages/lib-ethers/utils/deploy.ts]: Good point, although it is technically correct. it lacks clarity on what concerns the different deployment sets identification. Resolved in [feat: add pcv initialization in deployment scripts](https://github.com/Threshold-USD/dev/pull/102/commits/42f0ac098fdf83eef4a659e32dd25867d659e704)

**Review by vzotova — APPROVED** [review]

File: packages/lib-ethers/utils/deploy.ts [hunk 2]
```diff
@@ -11,14 +11,12 @@ import {
 interface IOwners {
   feePool: string
   bammOwner: string
-  legalEntity: string
   governorBravo: string
 }
 
 const contractOwners: IOwners = {
   feePool: "0x7095F0B91A1010c11820B4E263927835A4CF52c9",
```
Comment (cygnusv) [inline packages/lib-ethers/utils/deploy.ts:18]: This seems like a Liquity contract (https://etherscan.io/address/0x7095F0B91A1010c11820B4E263927835A4CF52c9). Is that correct?

File: packages/lib-ethers/utils/deploy.ts [hunk 3]
```diff
@@ -11,14 +11,12 @@ import {
...
-  bammOwner: "0x0000000000000000000000000000000000000001",
-  legalEntity: "0xf642Bd6A9F76294d86E99c2071cFE2Aa3B61fBDa",
+  bammOwner: "0x788c26aB09728A2f17F2372a4D65CcA184940E83",
```
Comment (cygnusv) [inline packages/lib-ethers/utils/deploy.ts:19]: What's this address?

**Review by vzotova — APPROVED** [review]

File: packages/lib-ethers/utils/deploy.ts (hunk 3 above)
Comment (evandrosaturnino) [reply packages/lib-ethers/utils/deploy.ts:19]: It's a 2 out of 2 multi-sig contract address, being one address of B. protocol owner contract, and the other one set for our side with a multi-sig address set with 5 committee volunteer members. More info can be seen at: https://discord.com/channels/866378471868727316/1157795257342230598/1157947513605734410

File: packages/lib-ethers/utils/deploy.ts (hunk 2 above)
Comment (evandrosaturnino) [reply packages/lib-ethers/utils/deploy.ts:18]: It's the feePool contract address provided by B. protocol, the same one used in Liquity BAMM contracts, more info can be seen here: https://docs.bprotocol.org/technical-documentation/smart-contracts/liquity and here: https://discord.com/channels/866378471868727316/918241759983710298/1156303650222768188

**Pull request merged**